runtimes/common/Cargo.toml

```diff
@@ -75,8 +68,7 @@ runtime-benchmarks = [
 ]
 std = [
   "attestation/std",
-  "parity-scale-codec/std",
-  "ctype/std",
+  "parity-scale-codec/std", # "ctype/std",
```

Comment:
There were a lot of unused imports. I removed them since the compiler would do this anyway. Comments are removed in: [10d9414](https://github.com/KILTprotocol/kilt-node/pull/527/commits/10d94148d24e423d574f1afd776cc79faccb9e09)